support-frontend/cloud-formation/cfn.yaml

```diff
@@ -278,8 +281,15 @@ Resources:
     Properties:
       AlarmActions:
       - !Sub arn:aws:sns:${AWS::Region}:${AWS::AccountId}:reader-revenue-dev
-      AlarmName: !Sub HIGH severity - No healthy instances for support-frontend in ${Stage}
-      AlarmDescription: Impact - Cannot sell any subscriptions or contributions products. Follow the process in https://docs.google.com/document/d/1_3El3cly9d7u_jPgTcRjLxmdG2e919zCLvmcFCLOYAk/edit
+      AlarmName: !Join
+        - ' '
+        - - !FindInMap [ StageVariables, !Ref Stage, AlarmUrgent ]
+          - !Ref 'Stage'
+          - 'no healthy instances for support-frontend'
+      AlarmDescription: !Join
+        - ' '
+        - - 'Impact - Cannot sell any subscriptions or contributions products.'
+          - !FindInMap [ StageVariables, !Ref Stage, AlarmProcess ]
```

Comment:
I was hoping storing the process link and header in one place would make it easier to read the cloudformation, but make your own mind up!